new_movies_tracker.py

```diff
@@ -117,5 +122,12 @@ def main():
     tracker = NewMoviesTracker(settings["movies"])
     tracker.process()
 
+
+def setup_logging():
+    logging_format = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+    logging.basicConfig(format=logging_format, level=logging.DEBUG)
```

Comment:
Does it really work? AFAIK, you instantiate logger before running setup_logging()